pkg/beholder/global.go

```diff
@@ -43,6 +43,10 @@ func GetEmitter() Emitter {
 	return GetClient().Emitter
 }
 
+func GetAuthenticator() *Authenticator {
+	return GetClient().Authenticator
+}
```

Comment:
Can we drop this now too? (and unexport `Authenticator`?)

Reply:
I think we still need this to be exported? Both are being used to retrieve the headers to pass to the LOOPP e.g. `beholder.GetAuthenticator().GetHeaders()`

Reply:
I think we should avoid that: https://github.com/smartcontractkit/chainlink/pull/15040#discussion_r1826309910